results/cmd/api/main.go

```diff
@@ -35,10 +35,20 @@ func main() {
 	// Connect to sqlite DB.
 	db, err := sql.Open("sqlite3", "./results.db")
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalf("failed to open the results.db: %v", err)
 	}
 	defer db.Close()
 	srv := &server{db: db}
+	database := srv.db
+
+	statement, err := database.Prepare("CREATE TABLE IF NOT EXISTS taskrun (logid binary(16) PRIMARY KEY, taskrunlog BLOB, uid INTEGER, name TEXT, namespace TEXT)")
```

Comment:
Since we have instructions to create the table in README, we should be able to remove this.

Reply:
done